src/prefect/client/client.py

```diff
@@ -875,6 +877,16 @@ def create_flow_run(
         if run_name is not None:
             inputs.update(flow_run_name=run_name)  # type: ignore
         res = self.graphql(create_mutation, variables=dict(input=inputs))
+
+        import requests
+
+        url = urljoin(self.agent_server, "/poke")
+        self.logger.debug("Poking agent process for new flow runs: %s", url)
+        try:
+            requests.get(url, timeout=0.25)
+        except requests.exceptions.RequestException as rex:
+            self.logger.info("Error poking agent process: %s", str(rex))
+
```

Comment:
Hmm I'm not sure about having this in the create_flow_run function. I think it would be better split out into a separate function or perhaps actually moved into some different utility. The Client is generally reserved for interacting with the backend API and the agent API is completely separate

Reply:
Yeah, we don't even guarantee that the agent api is running. We might not even have a method in prefect core for doing this, and leave poking the agent up to the users themselves. The agent itself may be behind a firewall, running on external infrastructure, behind a proxy with a path prefix, behind an authenticating proxy,  etc... There's too many complications here with how people may deploy prefect in production to make this worth exposing directly to users IMO.

Reply:
Thanks for your inputs @joshmeek @jcrist. I agree that it doesn't really makes sense to have client poke the agent, and should be left to the user on how they want to do it. 

I will make the necessary adjustments and resend the PR.